include/message_filters/message_traits.h

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
In the past when we copied code from other repos (without the file's history) we place a comment in the file pointing to where the file was copied from (e.g. https://github.com/ros2/turtlebot2_demo/blob/2017ddd2c5781434d9dbcb0c172d553e28f7ab02/depthimage_to_pointcloud2/include/depthimage_to_pointcloud2/depth_traits.hpp#L34)

Then adapting the header guards (as mentioned in https://github.com/ros2/message_filters/diffs#r206407588) to match the name of the new hosting package is IMO preferred but not a blocker for merging

Reply:
@tfoote did you see this comment ?

Reply:
Yeah I did, but I wanted to get this in since it was blocking our other steps. I'll open a new PR with the sources.

Reply:
https://github.com/ros2/message_filters/pull/6

Reply:
> Yeah I did, but I wanted to get this in since it was blocking our other steps. I'll open a new PR with the sources.


:+1: sounds good, just wanted to make sure it didnt get lost in the process.
Thanks for opening the follow-up PR!